lib/msal-node/src/client/ClientApplication.ts

```diff
@@ -412,10 +412,23 @@ export abstract class ClientApplication {
             : this.config.auth.azureCloudOptions;
 
         // using null assertion operator as we ensure that all config values have default values in buildConfiguration()
-        this.logger.verbose(
-            `building oauth client configuration with the authority: ${authority}`,
-            requestCorrelationId
-        );
+        this.logger.verbose("here-----------------------------------------------------------------------------------------------------------------");
```

Comment:
Can we stick to one format for logs? I do not think we had separators in general and if you want to add, would help if they are standardized.

Reply:
I think @trwalke meant to delete this

Reply:
Yes, not needed